include/sys/vnode.h

```diff
@@ -24,7 +24,7 @@ typedef struct componentname componentname_t;
 #define VNOVAL (-1)
 
 /* vnode access modes */
-typedef enum { VEXEC = 1, VWRITE = 2, VREAD = 4 } accmode_t;
+typedef enum { VEXEC = 1, VWRITE = 2, VREAD = 4, VADMIN = 8 } accmode_t;
```

Comment:
What does `VADMIN` mean? Also, why isn't it granted to root in `cred_can_access()`?

Reply:
`VADMIN` means "owner of the file". I am not sure if root shoud get VADMIN access. I will check that. Also we don't use it now so I will introduce it later.